.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build APK
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          push: false
+          build-args: |
+            flutter_version=latest
+            build_flavor=production
+
+      - name: Build and Release APK
+        uses: docker://docker.pkg.github.com/emprius/emprius-app-frontend:latest
+        env:
+          APK_OUTPUT_DIR: /emprius-APK/
+        with:
+          args: |
+            flutter packages upgrade &&
+            flutter build apk --release --flavor production
```

Comment:
This is seem correct. You should check what exact command of flutter are whe using here https://github.com/emprius/emprius-app-frontend/blob/main/misc/android_base.Dockerfile#L12 (we are not using `--release` I hope that the command is this)